src/slideseq/plot/plot_downsampling.py

```diff
@@ -105,9 +107,9 @@ def model_least_squares(params, *, r, data):
     ax = fig.add_axes([0.1, 0.1, 0.8, 0.8])
 
     for y, label, color in zip(
-        (y_100, y_80, y_60, y_40, y_20),
-        ("100", "80", "60", "40", "20"),
-        ("r", "b", "g", "k", "p"),
+            (y_100, y_80, y_60, y_40, y_20),
+            ("100", "80", "60", "40", "20"),
+            ("r", "b", "g", "k", "p"),
```

Comment:
Remove extra indentation
```suggestion
        (y_100, y_80, y_60, y_40, y_20),
        ("100", "80", "60", "40", "20"),
        ("r", "b", "g", "k", "p"),
```